extra-dev/packages/coq-itree-io/coq-itree-io.dev/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+maintainer: "Yishuai Li <yishuai@upenn.edu>"
+version: "dev"
+
+homepage: "https://github.com/Lysxia/coq-itree-io"
+dev-repo: "git+https://github.com/Lysxia/coq-itree-io.git"
+bug-reports: "https://github.com/Lysxia/coq-itree-io/issues"
+license: "MIT"
+
+synopsis: "Run interaction trees in IO"
+description: """
+Interpret itree in the IO monad of simple-io."""
+
+build: [make "-j%{jobs}%" ]
+install: [make "install"]
+depends: [
+  "coq" 
```

Comment:
Please add some kind of restriction on the Coq version here. If nothing else, use the lower bound from one of the dependencies.

Reply:
- Is it OK to remove `coq` dependency instead?
- Should we modify coq-community template to eliminate this situation?

Reply:
I think the solution is to explicitly add a Coq version bound through `supported_coq_versions` in your `meta.yml`. It's fine for Coq projects outside of our OPAM repos to not have any bounds (hence it is allowed by the templates). However, my view is that anyone who wants their Coq package in an official OPAM repo has to provide some kind of bound for Coq (a lower bound is enough, such as `>= 8.10`).